lib/core/src/obf.cpp

```diff
@@ -117,40 +119,30 @@ obfSetDebug( int opt ) {
 
 int
 obfiGetFilename( char *fileName ) {
-    char *envVar = NULL;
-
-    envVar = getRodsEnvAuthFileName();
-    if ( envVar != NULL && *envVar != '\0' ) {
-        strcpy( fileName, envVar );
-        return 0;
-    }
+    char* envVar{getRodsEnvAuthFileName()};
 
-#ifdef windows_platform
-    if ( ProcessType != CLIENT_PT ) {
```

Comment:
Was this line removed on purpose? Is it needed?

Reply:
It was intentional, thought we wanted to remove the windows code?
It shouldn't be needed to my knowledge, but maybe there's a reason otherwise?

Reply:
I see now. The PP else case doesn't check the `ProcessType`.

Carry on.